src/server/models/templates.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const quizTemplateSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  description: {
```

Comment:
Wouldn't it be easier to recommend any template based on tags and categories. Please add categories as attribute for optimised search.

Reply:
On it!